presto-product-tests/bin/run_on_docker.sh

```diff
@@ -218,27 +207,18 @@ else
 fi
 environment_compose up -d ${EXTERNAL_SERVICES}
 
-# start docker logs for the external services
-environment_compose logs --no-color -f ${EXTERNAL_SERVICES} &
-
 # start ldap container
 if [[ "$ENVIRONMENT" == "singlenode-ldap" ]]; then
   environment_compose up -d ldapserver
 fi
 
-HADOOP_LOGS_PID=$!
-
 # wait until hadoop processes is started
 retry check_hadoop
 stop_unnecessary_hadoop_services
 
 # start presto containers
 environment_compose up -d ${PRESTO_SERVICES}
 
-# start docker logs for presto containers
-environment_compose logs --no-color -f ${PRESTO_SERVICES} &
```

Comment:
The Presto logs are actually quite helpful for debugging issues -- I've used them to see if Presto OOMs and what exactly was happening when it did.